azure-pipelines.yml

```diff
@@ -166,8 +166,8 @@ stages:
             - task: DotNetCoreCLI@2
               displayName: Dotnet restore $(tfsProcessorSolution)
               env:
-                ARTIFACTORY_NUGET_USERNAME: $(ARTIFACTORY_PRIVATE_READER_USERNAME)
-                ARTIFACTORY_NUGET_PASSWORD: $(ARTIFACTORY_PRIVATE_READER_ACCESS_TOKEN)
+                ARTIFACTORY_USER: $(ARTIFACTORY_PRIVATE_READER_USERNAME)
+                ARTIFACTORY_PASSWORD: $(ARTIFACTORY_PRIVATE_READER_ACCESS_TOKEN)
```

Comment:
And likely, there's a new `NuGet.config` in the ITs that will need updating

Reply:
yeap - I was too fast. Thanks.